CMakeLists.txt

```diff
@@ -75,6 +76,7 @@ set_property(CACHE POLYFEM_THREADING PROPERTY STRINGS "CPP" "TBB" "NONE")
 option(POLYFEM_WITH_REMESHING "Uses VMTK for remeshing"                     OFF)
 option(POLYFEM_WITH_TESTS     "Build tests"                                 ON)
 option(POLYFEM_WITH_CLIPPER   "Use clipper, necessary for polygonal bases"  ON)
+# option(POLYFEM_WITH_MMG       "Build MMG utils for remeshing"               OFF)
```

Comment:
uncomment?

Reply:
MMG is currently disable in the main PolyFEM and unused (even in my changes). I just wanted to push my recipe to replace the old commented out CMake code.

Reply:
yes, but if the opt is off, it will remain unused but with the option to use it, no?